static/app/views/replays/detail/console/consoleMessage.tsx

```diff
@@ -123,8 +129,15 @@ const ConsoleTimestamp = styled(Common)<{isLast: boolean; level: string}>`
   cursor: pointer;
 `;
 
-const Icon = styled(Common)`
+const Icon = styled(Common)<{isActive: boolean}>`
   padding: ${space(0.5)} ${space(1)};
+  border-left: 4px solid
+    ${p =>
+      p.isActive
+        ? p.theme.active
+        : ['warning', 'error'].includes(p.level)
+        ? p.theme.alert[p.level].backgroundLight
```

Comment:
What about making it transparent?

Reply:
That works, no jumping and the background color is maintained.